.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,47 @@
+name: Docker Image
+on: 
+  push:
+    tags:
+      - '**'
+    branches:
+      - master
+  pull_request:
```

Comment:
Not sure if it makes sense to do a 1h build for each pull request

Reply:
Only pull requests changing something image related.
Later we can enable caching for PR builds while performing clean `master` and tag builds.
This PR does not do anything new here compared to what already was done on Travis.